project/OSGi.scala

```diff
@@ -23,7 +23,8 @@ object OSGi {
     OsgiKeys.privatePackage := Seq())
 
   val actor = osgiSettings ++ Seq(
-    OsgiKeys.exportPackage := Seq("akka*"),
+    OsgiKeys.bundle := OsgiKeys.bundle.dependsOn(compile in MultiReleaseJar).value,
+    OsgiKeys.exportPackage := Seq("akka*", "META-INF.versions.9.akka*"), // TODO: Refine this as it errors
```

Comment:
whoa that's weird; if it makes things work then cool I guess

Reply:
This is a bit crude. It puts the classfile in the right place in the jar. But from an OSGi bundle standpoint it's probably not legal - thus the non-fatal warning it spits out. Up to you if this is a blocker.